sbin/generate_grade_summaries.py

```diff
@@ -3,7 +3,7 @@
 Script to trigger the generate grade summaries.
 
```

Comment:
would be nice to add the documentation about the source argument in this file also?